spec/spec_helper.rb

```diff
@@ -1,13 +1,9 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
-ENV["RAILS_ENV"] ||= "test"
+ENV["RAILS_ENV"] ||= 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.